### PR TITLE
Create Matrix1dList widget for Tagging

### PR DESCRIPTION
**Pull request by tnagorra**

### CODE REVIEW

File: src/views/EditEntries/widgets/Matrix1dList/Matrix1dListOutput/index.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        // FIXME: just push attributeCell
+        return [
+            ...acc,
+            cell,
+        ];
+    },
+    [],
+);
+
+const getSelectedRowss = (rows, value) => {
```
Comment (AdityaKhatri) [inline src/views/EditEntries/widgets/Matrix1dList/Matrix1dListOutput/index.js]: Typoo

File: src/views/EditEntries/widgets/Matrix1dList/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+const emptyArray = [];
+const getRows = (widget = {}) => {
+    const { properties: { data: { rows = emptyArray } = {} } = {} } = widget;
+    return rows;
+};
+
+export default class Matrix1dListWidget extends React.PureComponent {
+    static valueKeyExtractor = d => d.key;
```
Comment (AdityaKhatri) [inline src/views/EditEntries/widgets/Matrix1dList/index.js]: After defaultProps

File: src/widgetComponents/Matrix1dListOutput/Row/Cell/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+export default class Cell extends React.PureComponent {
+    static propTypes = propTypes;
+
+    render() {
+        const { value } = this.props;
+
+        const marker = '•';
+
+        return (
+            <div className={styles.cell}>
```
Comment (frozenhelium) [inline src/widgetComponents/Matrix1dListOutput/Row/Cell/index.js]: Make this into new component (eg: ListItem)

**Review by frozenhelium — CHANGES_REQUESTED** [review]

File: src/widgetComponents/Matrix1dListOutput/index.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        // FIXME: just push attributeCell
+        return [
+            ...acc,
+            cell,
+        ];
+    },
+    [],
+);
+
+const getSelectedRowss = (rows, value) => {
```
Comment (frozenhelium) [inline src/widgetComponents/Matrix1dListOutput/index.js]: Possible typo

**Pull request merged**